formidling/src/main/java/no/nav/ung/sak/formidling/dokarkiv/DokArkivKlientImpl.java

```diff
@@ -0,0 +1,51 @@
+package no.nav.ung.sak.formidling.dokarkiv;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import jakarta.enterprise.context.Dependent;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.core.UriBuilder;
+import no.nav.k9.felles.integrasjon.rest.OidcRestClient;
+import no.nav.k9.felles.integrasjon.rest.ScopedRestIntegration;
+import no.nav.k9.felles.konfigurasjon.konfig.KonfigVerdi;
+import no.nav.ung.sak.formidling.dokarkiv.dto.OpprettJournalpostRequest;
+import no.nav.ung.sak.formidling.dokarkiv.dto.OpprettJournalpostResponse;
+
+/**
+ * https://confluence.adeo.no/display/BOA/opprettJournalpost
+ */
+@Dependent
+@ScopedRestIntegration(scopeKey = "DOKARKIV_SCOPE", defaultScope = "api://prod-fss.teamdokumenthandtering.dokarkiv/.default")
+public class DokArkivKlientImpl implements DokArkivKlient {
```

Comment:
Vi kan vurdere å flytte klientene her til k9-felles, men antagelig best å først får det ut i q for test

Reply:
ja, det var jeg litt usikker på fordi jeg lagde disse spesifikt for formidling og kun formidling som bruker dem, men jeg kan godt putte dem i felles. De er jo gjenbrukbare